packages/expand-button-react/src/ExpandButton.tsx

```diff
@@ -0,0 +1,49 @@
+import cx from "classnames";
+import React, { ReactNode } from "react";
+import { ScreenReaderOnly } from "@fremtind/jkl-core";
+import { ArrowVerticalAnimated } from "@fremtind/jkl-icons-react";
+
+export interface ExpandButtonProps {
+    children: ReactNode;
+    className?: string;
+    onClick?: React.MouseEventHandler<HTMLButtonElement>;
+    /**
+     * @default false
+     */
+    isExpanded?: boolean;
+    /**
+     * @default false
+     */
+    forceCompact?: boolean;
+    /**
+     * Skjul knappeteksten visuelt.
+     *
+     * @default false
+     */
+    srOnly?: boolean;
```

Comment:
For meg høres `srOnly` litt ut som om hele knappen blir skjult selv om det kun er teksten som skjules. Kanskje man kan ha et litt mer klargjørende navn? Hva med bare `hideText`/`hideContent`?